models/setup.py

```diff
@@ -1,15 +1,15 @@
-from pathlib import Path
 from sys import platform
 
 from setuptools import setup
 
-extra_deps = ['tensorflow-macos==2.10.0', 'tensorflow-metal==0.6.0'] if platform == "darwin" else ['tensorflow==2.10.0']
+extra_deps = ['tensorflow-macos==2.10.0',
+              'tensorflow-metal==0.6.0'] if platform == "darwin" else ['tensorflow==2.10.0']
 
-midi_dir = Path(__file__).resolve().parent.parent.joinpath('midi')
+# ../midi is not listed as requirements! It is the responsibility of `models` dependents to also install `../midi`
```

Comment:
**dependents** or **dependencies**?
tbh didn't know _dependents_ is a valid word

Reply:
I did mean _dependents_ (those that depend on). I will change the wording here.